backend/products/dao/insert.py

```diff
@@ -0,0 +1,47 @@
+import sys
+sys.path.append('..')
+from database.connection import *
+
+
+
+def product_insert(data):
+    '''
+    creates a table named product which has all the product details
+    A JSON file is given as input
+    the function connects to the database and adds data to the product table
+    '''
+    res=connectDB()
+    conn=res[0]
+    cur=res[1]
+    
+    try:
+        cur.execute("create table product(product_id text PRIMARY key,title text,description text,image_url text,price text,cat_id text)")
+    except:
+        return {"Status":400}
+    mapp = {}
+    count = 0
+
+    for i in data:
+        catLevel1=str(i['catlevel1Name'])
+        if catLevel1 not in mapp:
+            mapp[catLevel1] = count
+            count += 1
+    
+    for i in data:
```

Comment:
proper looping variable, instead of try-except you can use `i.get('productDescription', "")`